### ISSUE TITLE
Struct size incorrect when doing HiDPi through GTK3 env var

### ISSUE THREAD

**Issue by dongcarl** [taffybar/taffybar#264]
Previous discussion: https://github.com/travitch/taffybar/issues/179#issuecomment-377809032

1. Set `GDK_SCALE=2` per https://wiki.archlinux.org/index.php/HiDPI#GDK_3_.28GTK.2B_3.29
2. Start Xmonad with Taffybar (all default configs)
3. Start a terminal window
4. Notice that Taffybar scaled properly, but is obscured

**Comment by IvanMalison** [comment]
This sort of thing is actually going to be handled by gtk-strut see #268, so I think this will actually best be handled there.

**Comment by IvanMalison** [comment]
Right, gtk-strut just doesn't support this type of scaling. It would need to account for it in it's code

**Comment by IvanMalison** [comment]
Oh actually it looks like someone added a pr.to fix it ostensibly. Not sure if that be soon of gtk strut has been released though
